contracts/scripts/standard-bridge/DeployStandardBridge.s.sol

```diff
@@ -23,7 +23,7 @@ contract BridgeBase is Script {
     // AND initially fund the relayer, all on mev-commit chain.
     // This amount of ETH must be initially locked in the L1 gateway contract to ensure a 1:1 peg
     // between mev-commit chain ETH and L1 ETH.
-    uint256 public constant MEV_COMMIT_CHAIN_SETUP_COST = 1 ether + RELAYER_INITIAL_FUNDING;
+    uint256 public constant MEV_COMMIT_CHAIN_SETUP_COST = 10000 ether + RELAYER_INITIAL_FUNDING;
```

Comment:
Chain setup will not cost 10000 ether

> This amount of ETH must be initially locked in the L1 gateway contract to ensure a 1:1 peg between mev-commit chain ETH and L1 ETH.

10k eth is not feasible for us to lock in the L1 contract

Reply:
This is for testnet/devnet only.

Reply:
Could keep as-is if necessary, but ideally the allocation config is constant across all deployment types and bidders use the bridge in devnet